app/Livewire/Source/Github/Change.php

```
@@ -210,6 +210,8 @@ public function checkPermissions()
 
             GithubAppPermissionJob::dispatchSync($this->github_app);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Jobs\GithubAppPermissionJob' in method 'checkPermissions'. (undefined)

(StaticAccess)